dist/components/ui/msm-click-to-edit/msm-click-to-edit.scss

```diff
@@ -0,0 +1,37 @@
+.editable-text-container {
+    cursor: pointer;
+}
+
+.editable-text {
```

Comment:
Whenever you create something new (and don't extend bootstrap's classes), please use the `msm` prefix for the component's main class. `.msm-editable-text`.